wp_api/src/request.rs

```diff
@@ -151,9 +154,10 @@ impl WpNetworkRequestBody {
 }
 
 // Has custom `Debug` trait implementation
-#[derive(uniffi::Object)]
+#[derive(uniffi::Object, Clone)]
```

Comment:
I try to avoid deriving `Clone` as long as possible, so we don't accidentally abuse it. It's not the most costly `clone` since the header map is `Arc<T>`, but it's still pretty easy to avoid it, so I'd prefer if don't add it.
```suggestion
#[derive(uniffi::Object)]
```